sample-apps/go-sample-app/collection/client.go

```diff
@@ -2,42 +2,37 @@ package collection
 
 import (
 	"context"
+	"os"
 	"time"
 
 	"go.opentelemetry.io/contrib/propagators/aws/xray"
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
-	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric"
 	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
 	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
-	"go.opentelemetry.io/otel/metric/global"
-	semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
-	"google.golang.org/grpc"
-
-	"go.opentelemetry.io/otel/sdk/metric/aggregator/histogram"
-	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
-	processor "go.opentelemetry.io/otel/sdk/metric/processor/basic"
-	selector "go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
+
+	"go.opentelemetry.io/otel/sdk/metric"
 	"go.opentelemetry.io/otel/sdk/resource"
 	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 )
 
 var cfg = GetConfiguration()
 
-const grpcEndpoint = "0.0.0.0:4317"
-
 const serviceName = "go"
 
+var testingId = ""
+
 var tracer = otel.Tracer("github.com/aws-otel-commnunity/sample-apps/go-sample-app/collection")
 
 // Names for metric instruments
-const apiTimeAlive = "timeAlive"
-const apiCpuUsage = "cpuUsage"
-const apiTotalHeapSize = "totalHeapSize"
-const apiThreadsActive = "threadsActive"
-const apiTotalBytesSent = "totalBytesSent"
-const apiTotalApiRequests = "totalApiRequests"
-const apiLatencyTime = "latencyTime"
+const time_alive = "time_alive"
+const cpu_usage = "cpu_usage"
+const total_heap_size = "total_heap_size"
+const threads_active = "threads_active"
+const total_bytes_sent = "total_bytes_sent"
+const total_api_requests = "total_api_requests"
+const latency_time = "latency_time"
```

Comment:
Change the constant values but not their names.  Go always uses camelCase, never snake_case.

Reply:
I have changed the constant names back to camelCase